solidity/contracts/BorrowerOperations.sol

```diff
@@ -647,18 +645,11 @@ contract BorrowerOperations is
         _requireAtLeastMinNetDebt(vars.netDebt);
 
         // ICR is based on the composite debt, i.e. the requested amount + borrowing fee + gas comp.
-        vars.compositeDebt = _getCompositeDebt(vars.netDebt);
+        uint256 compositeDebt = vars.netDebt + MUSD_GAS_COMPENSATION;
```

Comment:
I would leave the call to `_getCompositeDebt` for consistency - we call it from another place and we still have `_getNetDebt` in `LiduityBase`. We have space for that after refactoring `LocalVariables_*`. This line only gives 0.009 KB optimization.

Side note, I am surprised there is an optimization to the size with this change, as I would expect the compiler to inline an `internal pure` function. It's not a guarantee though.